src/bin/ion/commands/beta/inspect.rs

```diff
@@ -498,7 +505,7 @@ impl<'a> IonInspector<'a> {
         // If it is a container, `inspect_level` will handle stepping into it and writing any
         // nested values.
         if !self.reader.ion_type().unwrap().is_container() {
-            self.hex_buffer.push_str(" ");
+            self.hex_buffer.push(' ');
```

Comment:
🗺️ All other changes in this file (`inspect.rs`) are tweaks to satisfy `clippy`.